src/services/docid-service.ts

```diff
@@ -125,9 +125,10 @@ async function parseXml(text: string, displayName: string, apiType: ItemType, gi
                 x.MemberType[0] === memberType
             );
 
-            const docId = methodOrCtor.MemberSignature.find((x: any) => x.$.Language === 'DocId').$.Value.substring(2);
-            // Replace the parentheses with *.
-            return { docId: docId.split('(')[0].concat('*') };
+            let docId = methodOrCtor.MemberSignature.find((x: any) => x.$.Language === 'DocId').$.Value.substring(2);
+            // Add * (and remove parens and double backticks, if present).
+            docId = docId.split('(')[0].split("``")[0];
```

Comment:
What if instead of a double back tick it's a single? What's the difference between a single generic type-parameter and a double?